test-gui.sh

```diff
@@ -36,9 +36,15 @@ echo $! > bridge.pid
 cd frontend
 yarn run webdriver-update --quiet > /dev/null 2>&1
 
+ENABLE_SERVICE_CATALOG="true"
+
 if [ $# -gt 0 ] && [ -n "$1" ];
 then
-  yarn run test-suite --suite "$1" --params.openshift true --params.servicecatalog true
+    if [ $# -gt 1 ] && [ -n "$2" ] && [ "$2" = "prow" ]
+    then
+      ENABLE_SERVICE_CATALOG="false"
+    fi
+  yarn run test-suite --suite "$1" --params.openshift true --params.servicecatalog $ENABLE_SERVICE_CATALOG
```

Comment:
We'll only need this logic for a short time, and it only adds a crud test for a single resource (ClusterServiceBroker). I would just leave `--params.servicecatalog` off here until we can get service catalog enabled in prow.